backend/infrahub/core/diff/data_check_synchronizer.py

```diff
@@ -22,11 +23,15 @@ def __init__(
         self.conflict_recorder = conflict_recorder
 
     async def synchronize(self, enriched_diff: EnrichedDiffRoot) -> list[Node]:
-        proposed_changes = await NodeManager.query(
-            db=self.db,
-            schema=InfrahubKind.PROPOSEDCHANGE,
-            filters={"source_branch": enriched_diff.diff_branch_name, "state": ProposedChangeState.OPEN},
-        )
+        try:
+            proposed_changes = await NodeManager.query(
+                db=self.db,
+                schema=InfrahubKind.PROPOSEDCHANGE,
+                filters={"source_branch": enriched_diff.diff_branch_name, "state": ProposedChangeState.OPEN},
+            )
+        except SchemaNotFoundError:
+            # if the CoreProposedChange schema does not exist, then there's nothing to do
+            proposed_changes = []
```

Comment:
with this try/except, this class depends on the `CoreProposedChange` schema, which we can count on in an actual deployment, but it causes problems in tests where that schema does not exist